packages/client/modules/userDashboard/components/CreditCardModal/CreditCardForm.tsx

```diff
@@ -149,6 +150,16 @@ const CreditCardForm = (props: Props) => {
         handleError(error.message, error.message)
         return
       }
+
+      const refetchVariables = {
+        orgId: orgId,
+        first: 3
+      }
+
+      if (invoiceListRefetch) {
+        invoiceListRefetch(refetchVariables)
```

Comment:
Nitpick: If possible, I'd prefer to encapsulate `invoiceListRefetch` invocation inside of the `onSuccess` callback.

Reply:
Thanks for the feedback! I went ahead and moved it.